docs/platforms/javascript/common/sourcemaps/troubleshooting_js/debug-ids/index.mdx

```diff
@@ -0,0 +1,74 @@
+
+This approach is Sentry's recommended and most reliable method for associating minified JavaScript files with their original source code, making it easier to trace errors back to their source. As a result, **creating a release is no longer required**.
+
+<Alert>
+
+If you want to learn more about the rationale behind Debug IDs, we suggest taking a look at our [engineering blog](https://sentry.engineering/blog/the-case-for-debug-ids).
+
+</Alert>
+
+## How Sentry uses Debug IDs
+
+To use Debug IDs, you need **version 7.47** or higher of one of our [JavaScript SDKs](https://github.com/getsentry/sentry-javascript). Once you're using a supported SDK, the process works as follows:
+
+1. **A globally unique Debug ID is generated**: During the build process a Debug ID is generated, which is globally unique and ideally deterministic (typically a UUID).
+
+2. **Debug IDs are embedded in minified files**: Each minified JavaScript file includes a special comment, such as `//# debugId=DEBUG_ID`, where `DEBUG_ID` is the unique identifier generated in the previous step.
+
+3. **The same Debug ID is added to source maps**: Source maps also include the same Debug ID as a new attribute, linking the minified JavaScript file to its corresponding source map.
+
+4. **Stack traces in Sentry link the Debug ID**: When an error occurs, Sentry's event contains the Debug ID, which allows Sentry to automatically link the error to the correct source map. This ensures the stack trace is resolved back to the original source code, providing you with clearer and more accurate debugging information—even if the JavaScript has been minified or transformed.
+
+### Why use Debug IDs to uniminify your code?
+
+Unminifying your code is essential for effective debugging. While there are other [legacy ways](https://docs.sentry.io/platforms/javascript/sourcemaps/troubleshooting_js/legacy-uploading-methods/) to achieve this at Sentry, using **Debug IDs are recommended by Sentry** as it is the most reliable and deterministic method. By unminifying your code, you get much clearer stack traces when errors occur. Instead of seeing obfuscated code, you'll see the exact code you originally wrote, making it far easier to understand and resolve issues. Below are examples that demonstrates the difference:
+
+#### Bad Stack Trace
+![bad stack trace example](./img/bad-stack-trace.png)
+
+#### Good Stack Trace
+![good stack trace example](./img/good-stack-trace.png)
+
+## Retention Policy
+
+Debug IDs have a retention period of _90 days_, using a _time to idle_ expiration mechanism.
+This means that uploaded Debug IDs are retained for as long as they are actively being used for event processing.
+Once an id has not been used to process incoming events for at least 90 days, it will automatically expire and be eligible for deletion.
```

Comment:
```suggestion
Once an ID has not been used to process incoming events for at least 90 days, it will automatically expire and be eligible for deletion.
```